packages/x-scheduler/src/primitives/utils/adapter/getAdapter.ts

```diff
@@ -0,0 +1,6 @@
+import { AdapterLuxon } from './AdapterLuxon';
+
+// TODO: Decide if we want to support several date libraries. If so, create a provider to avoid creating several instances of the adapter.
```

Comment:
Is there anything to decide?
I assumed that we are going to do this to keep this differentiator. 👍 
In any case, we can get away with this before we extract adapters. 👌

Reply:
AFAIK we can only keep it if the Base UI pickers do